packages/main/src/styles/InputIcon.module.css

```diff
@@ -0,0 +1 @@
+@import "../themes/InputIcon.css";
```

Comment:
I'd suggest importing and re-exporting this in `Input.module.css` (see my comment below)